src/main/kotlin/no/nav/familie/ef/mottak/service/OppgaveService.kt

```diff
@@ -203,7 +203,9 @@ class OppgaveService(private val integrasjonerClient: IntegrasjonerClient,
         val stønadType = dokumenttypeTilStønadType(søknad.dokumenttype) ?: return BehandlesAvApplikasjon.INFOTRYGD
         return if (finnesBehandlingINyLøsning(søknad.fnr, stønadType)) {
             BehandlesAvApplikasjon.EF_SAK
-        } else if (søknad.behandleINySaksbehandling && sakService.finnesIkkeIInfotrygd(søknad)) {
+        } else if (søknad.behandleINySaksbehandling &&
```

Comment:
Burde man nesten kunna sette sammen denne med den andre og kun sjekke
`sakService.finnesIkkeIInfotrygd(fnr, stønadType)` ? Eller blir det feil? Då sjekker man ikke behandleINySaksbehandling, men trenger vi den?

Reply:
Hadde blitt litt feil nå. Hadde åpnet opp for barnetilsyn/skolepenger i prod. Det er foreløpig `behandleINySaksbehandling` vi bruker til å featuretoggle, men den togglen kunne sikkert vært her. Foreslår vi tar en opprydding neste uke.